tests/cargo-kani/proptest-lib/fails.expected

```diff
@@ -0,0 +1,4 @@
+SUCCESS\
```

Comment:
We are only running `works_fine`, right? I don't understand why this file is being included.

Reply:
I don't think it has any use as of now. This file is just leftover from when I was experimenting.

Reply:
Fixed with d25e6d296c9bfb4c0e83171eeb5a6d75666d8b18